src/appMain/sdl_preloaded_pt.json

```diff
@@ -701,6 +704,7 @@
                             "deviceStatus",
                             "engineOilLife",
                             "engineTorque",
+                            "climateData",
```

Comment:
Do we need to include this param to "vehicle_data" section of preloadedPT? I see that earlier we did that (for "engineTorque" and "externalTemperature" params, for example)

Reply:
@ychernysheva I believe that it's not necessary, but I found the commit that says that we need to investigate it https://github.com/smartdevicelink/sdl_core/pull/3282#discussion_r438804647

Reply:
@ychernysheva Preloaded updated as it was agreed, please check a5189ac7a4622df31934e3ae164c63385d1c3cd8